modules/EnsEMBL/Web/Component/Gene/ComparaOrthologs.pm

```diff
@@ -0,0 +1,138 @@
+=head1 LICENSE
+
+Copyright [2009-2023] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package EnsEMBL::Web::Component::Gene::ComparaOrthologs;
+
+use strict;
+use warnings;
+
+sub create_gene_tree_links {
+  my $self = shift;
+  my $params = shift;
+
+  my $hub          = $self->hub;
+  my $species_defs = $hub->species_defs;
+  my $species = $params->{species};
+  my $stable_id = $params->{stable_id};
+  my $orthologue = $params->{orthologue};
+
```

Comment:
We could use the `cdb` parameter to know if we're dealing with Pan Compara data, allowing us to set the correct action below:
```suggestion
  my $cdb = $params->{cdb};

  my $is_pan = $cdb =~/compara_pan_ensembl/;
```

Reply:
Ah, sorry, I thought Pan Compara in the [original ensembl-webcode code](https://github.com/Ensembl/ensembl-webcode/blob/release/111/modules/EnsEMBL/Web/Component/Gene/ComparaOrthologs.pm#L300) were somehow related to strains. Reading the code again, I see now that I was mistaken. Are there Pan Compara trees on the Metazoa website (and can the orthologs table link to them)?

Reply:
Yes on both counts (e.g. [FBgn0014857 Pan Compara orthologues](http://wp-np2-1e.ebi.ac.uk:8440/Drosophila_melanogaster/Gene/Compara_Ortholog/pan_compara?g=FBgn0014857)), though it's only for select species (listed in [the Pan Compara species file](https://github.com/Ensembl/ensembl-compara/blob/release/112/conf/pan/additional_species.json#L122-L145)).